sdk/search/Microsoft.Azure.Search.Service/src/Customizations/Indexers/Models/FieldMappingFunction.Customization.cs

```diff
@@ -44,6 +44,31 @@ public partial class FieldMappingFunction
                     [nameof(useHttpServerUtilityUrlTokenEncode)] = useHttpServerUtilityUrlTokenEncode
                 });
 
+        /// <summary>
+        /// Creates a field mapping function that performs a simple URL-safe encoding of the input string.
+        /// It uses UTF-8 encoding and utilizes HttpUtility's UrlEncode method.
```

Comment:
Should we be leaking server-side implementation details here?

Reply:
Fair enough - I'll first update our documentation for field mapping functions (similar to base64encode and decode) and then refer to that link here.

I'll hold off on this PR, until the documentation is out at which point I'll send a new commit updating the docs

Reply:
On second thought, I think URL-encoding + specifying the encoding type should be generic enough and useful as-is and doesn't require serializing behind a documentation update - new commit incoming :)